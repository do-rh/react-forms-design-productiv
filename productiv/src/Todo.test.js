import { render, fireEvent } from "@testing-library/react";
import Todo from "./Todo";
const TEST_TODO = {
    title: "test title",
    description: "meow meow meow",
    priority: 1
};

it("renders without crashing", function () {
    render(<Todo {...TEST_TODO}/>)
});

it("matches snapshot", function () {
    const { container, debug } = render(<Todo {...TEST_TODO}/>)
    expect(container).toMatchSnapshot();
});

it("renders inputs correctly", function () {
    const { container, debug } = render(<Todo {...TEST_TODO}/>)
    expect(container).toContainHTML(TEST_TODO.title);
    expect(container).toContainHTML(TEST_TODO.description);
    expect(container).toContainHTML(`<small>Priority: ${TEST_TODO.priority}</small>`);
});