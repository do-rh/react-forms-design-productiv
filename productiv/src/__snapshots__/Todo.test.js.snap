// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Todo"
  >
    <div>
      <b>
        test title
      </b>
       
      <small>
        Priority: 
        1
      </small>
    </div>
    <div>
      <small>
        meow meow meow
      </small>
    </div>
  </div>
</div>
`;
